Makefile

```diff
@@ -0,0 +1,24 @@
+.PHONY: qa quality pylint pyroma
+
+quality:
+	isort sopel_slap
```

Comment:
This is a standalone tool, no? It's not in dev-requirements.

Reply:
Hm missed that one.

Reply:
Fixed.